{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 3, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "       TypeError: request() got an unexpected keyword argument 'rounds'\n", "test_name": "test_server_get_historical_1", "rerun": "0"}, "1": {"status": "FAIL", "message": "       TypeError: request() got an unexpected keyword argument 'rounds'\n", "test_name": "test_server_get_historical_2", "rerun": "0"}, "2": {"status": "FAIL", "message": "       TypeError: request() got an unexpected keyword argument 'rounds'\n", "test_name": "test_server_get_historical_interval_1", "rerun": "0"}}, "suite_name": "test.py"}}}, "date": "January 07, 2022", "start_time": 1641589722.4128363, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "3", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "3"}